next/src/components/sections/StarzLandingPageSection.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import LinkCard from '@/src/components/cards/LinkCard'
+import Banner from '@/src/components/common/Banner/Banner'
+import ResponsiveCarousel from '@/src/components/common/Carousel/ResponsiveCarousel'
+import SectionContainer from '@/src/components/layouts/SectionContainer'
+import { StarzLandingPageSectionFragment } from '@/src/services/graphql'
+import { generateImageSizes } from '@/src/utils/generateImageSizes'
+import { getLinkProps } from '@/src/utils/getLinkProps'
+import { isDefined } from '@/src/utils/isDefined'
+
+type StarzLandingPageSectionProps = { section: StarzLandingPageSectionFragment }
+
+const StarzLandingPageSection = ({ section }: StarzLandingPageSectionProps) => {
+  const { media, ...restBannerProps } = section.banner
+
+  const filteredCardLinks = section.cardLinks?.filter(isDefined) ?? []
+
+  return (
+    <SectionContainer className="py-6 lg:py-12">
+      <div className="flex flex-col gap-6 lg:gap-8">
+        <Banner imagePath={media.url} {...restBannerProps} className="flex-col-reverse" />
```

Comment:
I'm kind of convinced that we want to use this reversed approach everywhere. What do you think?